language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT_ID = "[your-project-id]"  # @param {type:"string"}
</pre><p>Could you change the line to the way I've written it above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>thanks - done</p>